File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocso/OcsoOCS1003P01LayPatInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocso
{
    [Serializable]
	public class OcsoOCS1003P01LayPatInfo
	{
		private String _gwa;
		private String _bunho;
		private String _doctor;
		private String _groupKey;
		private String _naewonYn;

		public String Gwa
		{
			get { return this._gwa; }
			set { this._gwa = value; }
		}

		public String Bunho
		{
			get { return this._bunho; }
			set { this._bunho = value; }
		}

		public String Doctor
		{
			get { return this._doctor; }
			set { this._doctor = value; }
		}

		public String GroupKey
		{
			get { return this._groupKey; }
			set { this._groupKey = value; }
		}

		public String NaewonYn
		{
			get { return this._naewonYn; }
			set { this._naewonYn = value; }
		}

		public OcsoOCS1003P01LayPatInfo() { }

		public OcsoOCS1003P01LayPatInfo(String gwa, String bunho, String doctor, String groupKey, String naewonYn)
		{
			this._gwa = gwa;
			this._bunho = bunho;
			this._doctor = doctor;
			this._groupKey = groupKey;
			this._naewonYn = naewonYn;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocso/OcsoOCS1003P01LayPatInfo.java
package nta.med.data.model.ihis.ocso;

public class OcsoOCS1003P01LayPatInfo {
	private String gwa;
    private String bunho;
    private String doctor;
    private String groupKey;
    private String naewonYn;
	public OcsoOCS1003P01LayPatInfo(String gwa, String bunho, String doctor,
			String groupKey, String naewonYn) {
		super();
		this.gwa = gwa;
		this.bunho = bunho;
		this.doctor = doctor;
		this.groupKey = groupKey;
		this.naewonYn = naewonYn;
	}
	public String getGwa() {
		return gwa;
	}
	public void setGwa(String gwa) {
		this.gwa = gwa;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public String getDoctor() {
		return doctor;
	}
	public void setDoctor(String doctor) {
		this.doctor = doctor;
	}
	public String getGroupKey() {
		return groupKey;
	}
	public void setGroupKey(String groupKey) {
		this.groupKey = groupKey;
	}
	public String getNaewonYn() {
		return naewonYn;
	}
	public void setNaewonYn(String naewonYn) {
		this.naewonYn = naewonYn;
	}
}
